### DISCUSSION TITLE
How GNPy can be converted to LOCAL HOST for changing the data rate and modulation format please assist

### DISCUSSION THREAD

**Discussion by NAIK131340** [Telecominfraproject/oopt-gnpy#466] [Q&A] [answered]
Dear @jktjkt, @ojnas @EstherLerouzic,
I’m working in Quality of transmission Estimation and Prediction Project in Optical Network and using GNPY for the same. I want to change to higher Data rate and modulation formats in transceiver OpenROADM MSA ver5.0 so how can I do that by changing it to local host so then how can I drag the GNPy project to local host so that i can make changes into it please give required documents for it the documents available aren't enough helping materials available for us to use GNPy. 
Thanking you so much

**Comment by jktjkt** [comment, marked as answer]
Welcome to GNPy!

> I want to change to higher Data rate and modulation formats in transceiver OpenROADM MSA ver5.0 so how can I do that

Once you've installed GNPy (or once you've started using a hosted/cloud version, for example), you have to [provide some input data](https://gnpy.readthedocs.io/en/master/concepts.html) to GNPy. GNPy ships with some sample data that are used by default. These are stored in a directory whose name you can print via running `gnpy-example-data`. In there, there's a bunch of "equipment config" input files. The most relevant for you is the `eqpt_config_openroadm_ver5.json` (also [here on GitHub](https://github.com/Telecominfraproject/oopt-gnpy/blob/master/gnpy/example-data/eqpt_config_openroadm_ver5.json)). You can copy that file to another location, make [any required changes](https://gnpy.readthedocs.io/en/master/extending.html#transponders), and then re-run GNPy using its `-e` option to point it to the file with your changes.

> by changing it to local host so then how can I drag the GNPy project to local host so that i can make changes

I'm sorry, but I don't understand this question. The [installation instructions](https://gnpy.readthedocs.io/en/master/install.html) outline some options.

**Comment by NAIK131340** [comment]
Hi JanKUndrat,
Thanks for you Help, In the eqpt_config_openroadms_ver5.json only upto 400 Gbit/s transmission used. I want to check it for 800 Gbit/s how can i do that.

**Reply by jktjkt** [reply]
OpenROADM v5 (the latest publicly available release) only defines transmission modes up to 400Gbps. If you're using a 800Gbps mode, it's not a mode whose definition has been standardized and made publicly available, which means that you will have to reach out to your system/HW vendor and obtain the performance numbers from them. With these numbers, you'll then be able to add a new modulation mode [as explained in the docs](https://gnpy.readthedocs.io/en/master/extending.html#transponders).
